consensus/votepower/roster.go

```diff
@@ -204,7 +204,10 @@ func Compute(subComm *shard.Committee, epoch *big.Int) (*Roster, error) {
 			ourPercentage = ourPercentage.Add(member.OverallPercent)
 		}
 
-		roster.Voters[staked[i].BlsPublicKey] = &member
+		// TODO: make sure external user's BLS key can be same as harmony's bls keys
+		if _, ok := roster.Voters[staked[i].BlsPublicKey]; !ok {
```

Comment:
do you need to print out a log  if the abnormal situation is found?

Reply:
done.